src/components/presets/midudev.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-use-before-define */
+
+import Inline from '../inline.macro'
+import { Image, Box, Link, Flex, Text } from './scope'
+
+const code = (
+  <Inline>
+    <>
+      <Link
+        href='https://fonts.googleapis.com/css2?family=Inter:wght@500;700&display=swap'
+        rel='stylesheet'
+      />
+      <Box as='header' sx={{}}>
```

Comment:
```suggestion
      <Box as='footer'>
```

Maybe that was your original intention? 😛

Reply:
it's worth it to move after the main component to be real footer!

Reply:
👍Moved to the footer! I started with the logo on the top of the image and I forgot to change it. :)